sycl/include/CL/sycl/detail/pi.h

```diff
@@ -245,9 +246,20 @@ static const uint8_t PI_DEVICE_BINARY_OFFLOAD_KIND_SYCL = 4;
 /// Target identification strings for
 /// pi_device_binary_struct.DeviceTargetSpec
 ///
+/// A device type represented by a particular target
+/// triple requires specific binary images. We need
+/// to map the image type onto the device target triple
+///
 #define PI_DEVICE_BINARY_TARGET_UNKNOWN "<unknown>"
+/// SPIR-V 32-bit image <-> 32-bit OpenCL device
```

Comment:
I think we also should clarify that "spir" is 32-bit OpenCL device.

Reply:
Does the line that your comment was applied to fulfill the purpose?

Reply:
Will update the comment clarifying the mapping